src/variants/R9MM/variant.h

```diff
@@ -122,9 +122,6 @@ extern "C" {
 /*----------------------------------------------------------------------------
  *        Arduino objects - C++ only
  *----------------------------------------------------------------------------*/
- 
- #define TIM_IRQ_PRIO       3
- #define EXTI_IRQ_PRIO      4
 
```

Comment:
^^ Note that just 9 lines above this are the exact same two lines

Reply:
And those lines were new for V1

Reply:
Unfortunately they came in as part of the humongous #505 PR, so it's hard to find out what the rationale was.